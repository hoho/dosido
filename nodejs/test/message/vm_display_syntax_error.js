'use strict';
var common = require('../common');
var assert = require('assert');
var vm = require('vm');

console.error('beginning');

vm.runInThisContext('var 5;', { filename: 'test.vm' });

console.error('end');
