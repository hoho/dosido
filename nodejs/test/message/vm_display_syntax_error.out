beginning

test.vm:1
var 5;
^^^
SyntaxError: Unexpected number
    at Object.exports.runInThisContext (vm.js:*)
    at Object.<anonymous> (*test*message*vm_display_syntax_error.js:*)
    at Module._compile (module.js:*)
    at Object.Module._extensions..js (module.js:*)
    at Module.load (module.js:*)
    at Function.Module._load (module.js:*)
    at Function.Module.runMain (module.js:*)
    at startup (node.js:*)
    at node.js:*
